code/modules/mob/mob.dm

```diff
@@ -1287,6 +1287,24 @@ var/list/slot_equipment_priority = list( \
 			var/mob/living/carbon/human/H = M
 			H.handle_regular_hud_updates()
 
+/mob/verb/rotate_chair() //To allow for actually sensible chair selection
```

Comment:
Which is mentioned in the OP. There isn't much I can do about it. How big of a problem is that? It just doesn't do anything if used when there isn't a chair adjacent to you.